protocol/src/codec/transaction.rs

```diff
@@ -119,10 +119,10 @@ impl Decodable for UnverifiedTransaction {
                 r.val_at(10)?
             },
             s:          if eth_tx_flag {
-                let tmp: U256 = r.val_at(10)?;
+                let tmp: U256 = r.val_at(11)?;
```

Comment:
what's the exact meaning of magic number 11 and 10? Why not replace them with meaningful words?